docs/buildah-bud.md

```diff
@@ -525,6 +525,7 @@ or it can be the path to a UTS namespace which is already in use by another
 process.
 
 **--volume, -v**[=*[HOST-DIR:CONTAINER-DIR[:OPTIONS]]*]
+**--volume, -v**[=*[overlay=HOST-DIR:CONTAINER-DIR[:OPTIONS]]*]
```

Comment:
Not wild about overloading this,  but the "overlay=" tag makes me feel better about it.